tests/bpf2c_tests/raw_bpf.cpp

```diff
@@ -117,20 +116,20 @@ run_test(const std::string& data_file)
     c_file.flush();
     c_file.close();
 
-    std::string compile_command = std::string(CC " " CXXFLAG " -I.." SEPERATOR ".." SEPERATOR "include ") +
-                                  std::string(prefix) + std::string(".c ") + std::string(" bpf_test.cpp >") +
-                                  std::string(prefix) + std::string(".log -o ") + std::string(prefix) +
-                                  std::string(EXT);
+    std::string compile_command = cc + std::string(" ") + cxxflags +
+                                  std::string(" -I.." SEPARATOR ".." SEPARATOR "include ") + std::string(prefix) +
+                                  std::string(".c ") + std::string(" bpf_test.cpp >") + std::string(prefix) +
+                                  std::string(".log");
     REQUIRE(system(compile_command.c_str()) == 0);
-    std::string test_command = std::string("." SEPERATOR) + std::string(prefix) + std::string(EXT) + std::string(" ") +
+    std::string test_command = std::string("." SEPARATOR) + std::string(prefix) + std::string(" ") +
                                std::string(result) + std::string(" \"") + std::string(mem) + std::string("\"");
     REQUIRE(system(test_command.c_str()) == 0);
 }
 
 #define DECLARE_TEST(FILE)                                                                                     \
     TEST_CASE(FILE, "[raw_bpf_code_gen]")                                                                      \
     {                                                                                                          \
-        run_test(".." SEPERATOR ".." SEPERATOR "external" SEPERATOR "ubpf" SEPERATOR "tests" SEPERATOR "" FILE \
+        run_test(".." SEPARATOR ".." SEPARATOR "external" SEPARATOR "ubpf" SEPARATOR "tests" SEPARATOR "" FILE \
```

Comment:
So this assumes the externel\ubpf directory is in this relative position from the actual test bits. So does this mean this test has to run in the same runner that is executing the build task? Also is it possible for the YML to pass the path to the directory?

Reply:
Agreed. We should copy the .data files to the output folder and remove the dependency at run time.